src/dbt/kipptaf/models/performance_management/intermediate/int_performance_management__overall_scores.sql

```diff
@@ -82,11 +112,13 @@ with
             and form_term in ('PM2', 'PM3')
             and rn_submission = 1
             and overall_score is not null
-        group by internal_id, academic_year
+            and academic_year >= 2023

```

Comment:
why is 2023 hardcoded?

Reply:
2023 is first year coming from SMG data directly, past years are in the archive source tables